src/config/dynamic-theme-fixes.config

```diff
@@ -8136,6 +8136,9 @@ web.whatsapp.com
 INVERT
 .selectable-text > span._3PxOr
 span[data-icon="audio-download"]
+span[dir="ltr"] > ._2ezGC
```

Comment:
Please use
```CSS
.invisible-space > span
```
We can't use a dir="ltr" as e.g. Arabic systems will have dir="rlt" and whatsapp has a known behaviour whereby the selectors are generated again every ~1 week.